src/services/iasql.ts

```diff
@@ -496,6 +498,15 @@ export async function apply(dbId: string, dryRun: boolean, ormOpt?: TypeormWrapp
                 }
               }));
             }
+            return outArr;
+          })
+          .flat(9001);
+        const reversePromiseGenerators = records
```

Comment:
Nice splitting the deletes from the creates that way. Update-as-Replace is in a weird spot here, but I think it should eventually converge since the core of the algo is still the same.